app/views/collaborations/_form.html.erb

```diff
@@ -13,30 +13,18 @@
   <% end %>
 
   <p>This is the owner:
-<%=  @wiky.user.email %></p>
+  <%=  @wiky.user.email %></p>
 
   <strong>Delete Collaborators</strong><br>
   <table class= "table table-striped">
-    
-
-
-  <% @user_ids_array.each do |user_id| %>
-  
-   
-    <tr>
-     <td><%= User.find(user_id).email %></td> 
-     <td><%= link_to('Destroy', wiky_collaboration_path(@wiky, (
-      @wiky.collaborations.where(user_id: user_id).pluck(:id))), 
-      :method => 'delete', controller: :collaborations)  %></td>
-    </tr>
-   
-
-  <% end %> 
- </table>
-
-
-
-
+    <% @collaborations.reload.each do |collaboration| %>
```

Comment:
If I don't write reload, for some reason I get <undefined method `email'
for nil:NilClass>.
Also if i write <%= @collaborations.first.user.email %> It does work
without reloading. I am trying to figure it out.

On Sat, Feb 14, 2015 at 7:57 AM, Eliot Sykes <notifications@github.com>
wrote:

> In app/views/collaborations/_form.html.erb
> <https://github.com/LuisMenjivar/wiki/pull/55#discussion_r24714295>:
>
> > -
> > -    <tr>
> > -     <td><%= User.find(user_id).email %></td>
> > -     <td><%= link_to('Destroy', wiky_collaboration_path(@wiky, (
> > -      @wiky.collaborations.where(user_id: user_id).pluck(:id))),
> > -      :method => 'delete', controller: :collaborations)  %></td>
> > -    </tr>
> > -
> > -
> > -  <% end %>
> > - </table>
> > -
> > -
> > -
> > -
> > +    <% @collaborations.reload.each do |collaboration| %>
>
> @LuisMenjivar <https://github.com/LuisMenjivar>
>
> I'm a little surprised to see .reload here, @collaborations.each do ...
> would normally be enough.
>
> What problem does the reload call here solve?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/LuisMenjivar/wiki/pull/55/files#r24714295>.
>